lib/jsonapi/authorization/authorizing_processor.rb

```diff
@@ -216,6 +224,10 @@ def authorize_remove_to_one_relationship
         authorizer.remove_to_one_relationship(source_record, relationship_type)
       end
 
+      def authorize_replace_polymorphic_to_one_relationship
+        raise NotImplementedError
```

Comment:
Actually, I think we'd like to skip raising this here and figure out what to do with the polymorphic relationship scenario outside of this PR. This way we wouldn't add too much noise to this upgrade path.

Would you like to remove this callback from the processor?